usecases/config/environment.go

```diff
@@ -380,7 +380,7 @@ func FromEnv(config *Config) error {
 		return err
 	}
 
-	if v := os.Getenv("REPLICATION_FORCE_OBJECT_DELETION_CONFLICT_RESOLUTION"); v != "" {
+	if v := os.Getenv("REPLICATION_FORCE_DELETION_STRATEGY"); v != "" {
```

Comment:
should we just do REPLICATION_DELETION_STRATEGY ?

Reply:
ahh, wasn't sure what the `force` was standing for - makes sense! Take or leave it :)